src/main/java/frc/team3128/subsystems/Shooter.java

```diff
@@ -227,16 +228,7 @@ public void simulationPeriodic() {
 
     public double calculateMotorVelocityFromDist(double dist) {
 
-        // dist += (13 - Robot.voltageRollingAvg) * 7;
-
-        return 0.00971 * Math.pow(dist, 3) - 0.289 * Math.pow(dist, 2) - 52.17 * dist + 5196 + 600;
-        // double rpm;
-        // if (dist < 78) {
-        //     rpm = 17.7 * dist + 2187;   
-        // } else {
-        //     rpm = -4.54 * Math.pow(dist, 2) + 917 * dist - 40328;
-        // }
-        // return rpm + 1000;
+        return -1.43648019e-3*dist*dist*dist + 4.42551199e-1*dist*dist - 3.02450570e1*dist + 3.16957933e3 - 100;
```

Comment:
Likely need to figure out if we need any offset for what the eqn. gives during driver practice tomorrow